core/src/main/java/org/mapfish/print/servlet/job/impl/ThreadPoolJobManager.java

```diff
@@ -628,8 +635,12 @@ public void run() {
         }
       } catch (javax.persistence.PessimisticLockException e) {
         // Ignore error on pessimistic locking
+        unhealthyCountersHealthCheck.recordUnhealthyProblem(
+            getClass().getSimpleName(), "ignorePessimisticLockException");
       } catch (RuntimeException t) {
         LOGGER.error("Error while polling/updating registry", t);
+        unhealthyCountersHealthCheck.recordUnhealthyProblem(
+            getClass().getSimpleName(), "exceptionWhilePollingRegistry");
```

Comment:
pollingRegistryError

Reply:
As Error has a different connotation in the try/catch world of java, how about pollingRegistryProblem or pollingRegistryIssue ? 
I suggest we use a word like Issue or Problem (as nothing is done in the code to address the exception.
And a word like Failure when something is done after identifying something went wrong.

Reply:
+1 for pollingRegistryIssue :-)